### ISSUE TITLE
Crashes after short while (Plasma wayland Kubuntu)

### ISSUE THREAD

**Issue by agrrr3** [flathub/com.umlet.Umlet#1]
Installed this as --user on a Ubuntu 22.04.1 LTS Kubuntu with Plasma, in a wayland session.
If I run the flatpak everything looks fine. And after a short while  the GUI crashes without any warning or error (but the process does not terminate).

The last thing I see is:
```
$ flatpak run --user com.umlet.Umlet 
log4j:WARN No appenders could be found for logger (com.baselet.control.enums.Program).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```
If I try to rerun using the same command, I get the same prompt, but no GUI at all.

Changing the session to Plasma (X11) works, i can use Umlet without it crashing.

Also the flatpak works without issue on one of my desktop machines with debian bullseye in Gnome session on (if I am not mistaken) wayland.

Note that this must be happen on xwayland, as the flatpak/java awt does not support native wayland.

**Comment by agrrr3** [comment]
Note also that the Plasma (X11) umlet crashes if I switch to another session and back again. In this case I can restart without issue though.

Switching or locking sessions is prone to misbehave on this kubuntu laptop so I am not sure this happens for other people to.

**Comment by zocker-160** [comment]
Thanks for reporting this, I did not notice, since I am running X11 myself.

It would be interesting to know, if this also happens when running the "native" .jar file or if this is something related to Flatpak.

**Comment by agrrr3** [comment]
Just tried the flatpak from the debian system in Plasma (Wayland). Works also without issue.
(just the native firefox behaves weird in that session though; has like tons of delay; whaaat?).

**Comment by agrrr3** [comment]
so whatever this is causing, it has to (also) to do with either the OS (kubuntu vs debian), my configuration or my hardware. i probably first try running another flatpak based on the java sdk than trying make java work native. any suggestions?

**Comment by zocker-160** [comment]
Maybe switching the java image to version 17 could help?
I can create a beta version for you to test.

How long does it usually take until it crashes? Just want to know if it is minutes or hours.

**Comment by agrrr3** [comment]
> Maybe switching the java image to version 17 could help? I can create a beta version for you to test.

yes, maybe lets try that. i tried org.briarproject.Briar which uses the version 17 and does not crash here. 

That also differs a little bit, it sets the JAVA_HOME in the openjdk module and finish-args (https://github.com/flathub/com.umlet.Umlet/blob/master/com.umlet.Umlet.yml); i expect those are not necessary as the java program clearly runs.

> 
> How long does it usually take until it crashes? Just want to know if it is minutes or hours.

usually less than a minute

**Comment by zocker-160** [comment]
@agrrr3 I created a PR to the beta branch with openjdk17, could you please try that out and see if there is any difference?

https://github.com/flathub/com.umlet.Umlet/pull/2#issuecomment-1329051081
